vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -88,7 +96,18 @@ The `conference-admin` role should grant a user permission to delete Conferences
 3. Modify the `deleteConference` method by preceding it with the following annotation:
 ```java{1}
     @PreAuthorize("hasAuthority('conference-admin')")
-    public ResponseEntity<Void> deleteConference(@PathVariable Long id)
+```
+The resulting code section should looke similar to this:
+``` java{1}
+@DeleteMapping("/conferences/{id}")
```

Comment:
Java doesn't care about the indentation but reads better if the annotations are aligned
```suggestion
   @DeleteMapping("/conferences/{id}")
```